### PR TITLE
[10.0][IMP] account_financial_report: adapt the wizards to multicompany

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This is a backport of https://github.com/OCA/account-financial-reporting/pull/489

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@alexis-via This is the backport of #489. Can you please review?

**Comment by JordiBForgeFlow** [comment]
Requires #498 to be merged and backported to this PR.

**Review by alexis-via — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@Cedric-Pigeon Can you please review this PR? Would be the next to merge after your PR is merged.

File: account_financial_report_qweb/report/abstract_report_xlsx.py [hunk 1]
```diff
@@ -360,7 +360,7 @@ def _get_currency_amt_header_format(self, line_object):
     def _generate_report_content(self, workbook, report):
         pass
 
-    def _get_report_name(self):
+    def _get_report_name(self, objects):
```
Comment (Cedric-Pigeon) [inline account_financial_report_qweb/report/abstract_report_xlsx.py:363]: wouldn't be better to put a better name than 'objects' ? Why not calling it report ?

File: account_financial_report_qweb/report/aged_partner_balance_xlsx.py [hunk 2]
```diff
@@ -15,8 +15,10 @@ def __init__(self, name, table, rml=False, parser=False, header=True,
         super(AgedPartnerBalanceXslx, self).__init__(
             name, table, rml, parser, header, store)
 
-    def _get_report_name(self):
-        return _('Aged Partner Balance')
+    def _get_report_name(self, objects):
+        report = objects
```
Comment (Cedric-Pigeon) [inline account_financial_report_qweb/report/aged_partner_balance_xlsx.py]: The renaming of the parameter would avoid this line

**Review by Cedric-Pigeon — CHANGES_REQUESTED** [review]

File: account_financial_report_qweb/report/aged_partner_balance_xlsx.py (hunk 2 above)
Comment (JordiBForgeFlow) [reply account_financial_report_qweb/report/aged_partner_balance_xlsx.py]: @Cedric-Pigeon I have fixed according to your comments.

**Review by Cedric-Pigeon — APPROVED** [review]

**Pull request merged**